src/test/java/com/at/stepdefinitions/Put.java

```diff
@@ -40,13 +40,18 @@ public void create_body_for_a_new_user(){
         jsonBodyRequest.put("email", lorem.getFirstName().toLowerCase()+"@gmail.com");
         jsonBodyRequest.put("password", lorem.getFirstName().toLowerCase()+"12345678");
         jsonBodyRequest.put("status", 0);
+        
         base.requestBody = jsonBodyRequest.toString();
-        base.apiResource = "/api/v1/users";
+        for(ApiPath a: ApiPath.values()) {
```

Comment:
Did you try using ApiPath.updating_users.getApiPath(); instead of using the for statement?